### PR TITLE
Remove `ORDER BY` statement

### PR DESCRIPTION
**Pull request by LukasReschke**
SQLite seems not to work here with a `ORDER BY` statement. - This prevented updates from previous versions.

@VicDeo Please review.

### CODE REVIEW

**Comment by karlitschek** [comment]
:+1:

**Comment by nickvergessen** [comment]
:+1:

File: appinfo/preupdate.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	$query = OC_DB::prepare( 'SELECT COUNT(*) AS `count`, `fileid` FROM `*PREFIX*files_antivirus` GROUP BY `fileid` HAVING COUNT(*) > 1' );
 	$result = $query->execute();
 	while( $row = $result->fetchRow()) {
-		$deleteQuery = OC_DB::prepare('DELETE FROM `*PREFIX*files_antivirus` WHERE `fileid` = ? ORDER BY `check_time` ASC', $row['count']-1);
+		$deleteQuery = OC_DB::prepare('DELETE FROM `*PREFIX*files_antivirus` WHERE `fileid` = ?', $row['count']-1);
```
Comment (nickvergessen) [inline appinfo/preupdate.php]: What kind of hack is `fileid = count -1` this makes NO sense at all, unless its explained in a comment.

Comment (LukasReschke) [inline appinfo/preupdate.php]: I certainly don't know. – I think it removes all entries except one… - There was no PK on this table before.

Comment (VicDeo) [inline appinfo/preupdate.php]: @LukasReschke correct. Adding a PK it appeared that table might have several scan results if the app is old enough. This cleans up scan results except the very last.

Comment (nickvergessen) [inline appinfo/preupdate.php]: Okay then

**Pull request merged**